cardano-cli/src/Cardano/CLI/Shelley/Commands.hs

```diff
@@ -238,14 +239,14 @@ data TransactionCmd
   | TxCalculateMinFee
       TxBodyFile
       (Maybe NetworkId)
-      ProtocolParamsFile
+      ProtocolParamsSource
```

Comment:
Likewise here, this is an offline command.

Reply:
You can't use `ProtocolParamsSource` here because it's an offline command i.e you do not have to run a node to use it.